src/main/java/com/google/devtools/build/lib/rules/cpp/CppOptions.java

```diff
@@ -305,6 +305,18 @@ public LipoModeConverter() {
   public boolean processHeadersInDependencies;
 
   @Option(
+    name = "experimental_enable_implicit_headermaps",
+    defaultValue = "false",
+    category = "semantics",
+    documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
+    effectTags = {OptionEffectTag.UNKNOWN},
+    help =
+        "When building a target //a:a, generate a header map containing all of the headrs that //a:a depends "
+            + "on (if header processing is enabled for the toolchain)."
```

Comment:
I'm confused, where is 'header processing is enabled' checked?

Reply:
Ah I mean't `header maps` - this feature is enabled with `experimental_enable_implicit_headermaps`. Perhaps the option should be explicitly called out here?